guides/wide-ep-lws/README.md

```diff
@@ -107,7 +107,9 @@ Select the provider-specific Helm command using the tabs below.
 
 ### Deploy Gateway and HTTPRoute
 
-Choose the gateway manifest that matches your environment.
+This guide uses custom gateway manifests in `./manifests/gateway/`. For standard deployments, you can use the [gateway recipe](../../recipes/gateway/README.md#install-gateway-and-httproute) directly.
+
+Choose the gateway manifest that matches your environment:
```

Comment:
For this guide, we should reference the gateway recipes directly and remove the gateway/httproute manifests.